common/resource.go

```diff
@@ -16,15 +16,17 @@ import (
 
 // Resource aims to simplify things like error & deleted entities handling
 type Resource struct {
-	Create         func(ctx context.Context, d *schema.ResourceData, c *DatabricksClient) error
-	Read           func(ctx context.Context, d *schema.ResourceData, c *DatabricksClient) error
-	Update         func(ctx context.Context, d *schema.ResourceData, c *DatabricksClient) error
-	Delete         func(ctx context.Context, d *schema.ResourceData, c *DatabricksClient) error
-	CustomizeDiff  func(ctx context.Context, d *schema.ResourceDiff) error
-	StateUpgraders []schema.StateUpgrader
-	Schema         map[string]*schema.Schema
-	SchemaVersion  int
-	Timeouts       *schema.ResourceTimeout
+	Create             func(ctx context.Context, d *schema.ResourceData, c *DatabricksClient) error
+	Read               func(ctx context.Context, d *schema.ResourceData, c *DatabricksClient) error
+	Update             func(ctx context.Context, d *schema.ResourceData, c *DatabricksClient) error
+	Delete             func(ctx context.Context, d *schema.ResourceData, c *DatabricksClient) error
+	CustomizeDiff      func(ctx context.Context, d *schema.ResourceDiff) error
+	StateUpgraders     []schema.StateUpgrader
+	Schema             map[string]*schema.Schema
+	SchemaVersion      int
+	Timeouts           *schema.ResourceTimeout
+	DeprecationMessage string
+	Importer           *schema.ResourceImporter
```

Comment:
These are the only fields from schema.Resource that need to be added to this Resource structure.